GliaWidgets/BundleManaging/BundleManaging.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+struct BundleManaging {
```

Comment:
I would add documentation here stating why we introduce this struct, because this approach may be foreign to some of the maintainers.

Reply:
Good move by the way 🙌